server/tapeuse.go

```diff
@@ -0,0 +1,64 @@
+package server
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+// A FixityDB tracks the information the fixity service needs to know what
+
+func (s *RESTServer) EnableTapeUse() {
+	log.Println("Enabling Bendo Tape Use")
+	s.TapeUsed = true
+	s.Items.SetUseStore(true)
+}
+
+func (s *RESTServer) DisableTapeUse() {
+	log.Println("Disabling Bendo Tape Use")
+	s.TapeUsed = false
+	s.Items.SetUseStore(false)
+}
+
+func (s *RESTServer) StatusTapeUse() bool {
+	return s.TapeUsed
+}
```

Comment:
OK

On Thu, Aug 25, 2016 at 12:41 PM, Don Brower <notifications@github.com>
wrote:

> In server/tapeuse.go
> <https://github.com/ndlib/bendo/pull/136#discussion_r76279694>:
>
> > +
> > +func (s *RESTServer) EnableTapeUse() {
> > +	log.Println("Enabling Bendo Tape Use")
> > +	s.TapeUsed = true
> > +	s.Items.SetUseStore(true)
> > +}
> > +
> > +func (s *RESTServer) DisableTapeUse() {
> > +	log.Println("Disabling Bendo Tape Use")
> > +	s.TapeUsed = false
> > +	s.Items.SetUseStore(false)
> > +}
> > +
> > +func (s *RESTServer) StatusTapeUse() bool {
> > +	return s.TapeUsed
> > +}
>
> remove this function and access the variable directly.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/bendo/pull/136/files/116b45889c785d5598950eba80d6730d33f98945#r76279694>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTFofZcSmUr-Tt3vNyqbUHGdcuLkzks5qjcWogaJpZM4JsckT>
> .
>